src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -200,6 +200,9 @@ registerCodeFix({
                     else if (canDeleteEntireVariableStatement(sourceFile, token)) {
                         deleteEntireVariableStatement(changes, sourceFile, token.parent as VariableDeclarationList);
                     }
+                    else if (isIdentifier(token) && isFunctionDeclaration(token.parent)) {
+                        deleteFunctionLikeDeclaration(changes, sourceFile, token.parent as FunctionLikeDeclaration);
+                    }

```

Comment:
this just follows the pattern used by the single-codefix "branch" here:
https://github.com/microsoft/TypeScript/blob/39b3aa0270edca1d07392b839b3c17cff7a0fdca/src/services/codefixes/fixUnusedIdentifier.ts#L127-L135